deployment/config.yaml

```diff
@@ -14,15 +14,15 @@
 
 project:
   dev: datcom-mixer-staging # datacommons-website-dev
-  staging: datacommons-website-staging
+  staging: dc-test-1-298300
```

Comment:
Should we re-make a new project with the name we'd like? Or add a comment to update it.

Reply:
that's the plan, added TODO